crates/hc_sandbox/tests/cli.rs

```diff
@@ -120,12 +120,10 @@ async fn get_app_info(admin_port: u16, installed_app_id: InstalledAppId, port: u
         .await
         .unwrap();
 
-    let request = AppRequest::AppInfo {
-        installed_app_id: "Stub".to_string(),
-    };
+    let request = AppRequest::AppInfo;
     let response = app_tx.request(request);
     let r: AppResponse = check_timeout(response).await;
-    assert_matches!(r, AppResponse::AppInfo(None));
+    assert_matches!(r, AppResponse::AppInfo(Some(_)));
```

Comment:
Can no make a request for an app that doesn't exist, so we get the info back for the app we've opened a connection for